fubectl.source

```diff
@@ -294,3 +315,17 @@ khelp() {
         grep -E '^# \[.+\]' "${BASH_SOURCE[0]}"
     fi
 }
+
+#### CUSTOM TOBI
```

Comment:
@toschneck is this an accident or you are willing to include this too?

Reply:
@kron4eg fixed it